bundles/org.openhab.binding.mystrom/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mystrom"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="mystromplug">
+		<label>myStrom Smart Plug</label>
+		<description>Controls the myStrom smart plug</description>
+
+		<channels>
+			<channel id="switch" typeId="switch-channel"/>
```

Comment:
You could use the system type. Then, you don't need to specify your own Channel type.
```suggestion
			<channel id="switch" typeId="system.power"/>
```

Reply:
done